index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Personal Portfolio</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+    <nav class ="main-nav">
+        <div class="logo">
+        <a href="#"><p>Worash Aboche</p></a>  
+        </div>
```

Comment:
- [ ] Please design should have  margins (top, bottom, left, right) defined in the design

| Your design | Figma design |
| ------------- | --------------  |
| ![image](https://user-images.githubusercontent.com/74744787/164341197-a0b20c25-b50a-4676-bd1b-2b1dc48cbb8f.png) | ![image](https://user-images.githubusercontent.com/74744787/164341336-b2797fd0-c3e6-4e74-83c0-a8718b133b0a.png) |